docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: "3.9"
+services:
+  web-dev:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: dev
+    container_name: spooky-list__web-dev
+    restart: always
+    volumes:
+      - ./:/app
+      - /app/node_modules
+      - /app/.next
+    ports:
+      - "3000:3000"
+  web-prod:
```

Comment:
this lets us run a production version of our app, which will be much faster. this is what we'll deploy.